index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
       <button type="button">Start Collaboration</button>
     </div>
 
-    <img src="./images/geometry.svg" id="geometry-image" alt="geometry">
+    <img src="./images/geometry.svg" alt="geometry">
+  </section>
+
+  <section id="Works">
+    <h2>Projects</h2>
+
+    <div class="project-card-section">
+      <div class="project-card">
+        <img src="./images/project-image.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
```

Comment:
- [ ] Kindly note that your app still overflows on the said mobile screens(which is a minimum of 375px), which shouldn't be, the device used in testing the app is 390px.

![mobile (20)](https://github.com/donice/Portfolio-Microverse/assets/41163571/c6d78651-6379-4155-a43a-9ac9904c238b)

Reply:
@AdnanOlarmmi  please how many hours do I have left?

Reply:
@donice, till 12am Microverse time... I guess you've been informed prior